granian/__main__.py

```diff
@@ -0,0 +1,4 @@
+from granian.cli import cli
+
+if __name__ == "__main__":
+    cli()
```

Comment:
Isn't the if redundant here? Probably this can be changed into just:

```suggestion
cli()
```